front/temporal/labs/activities.ts

```diff
@@ -268,8 +269,63 @@ export async function processTranscriptActivity(
     return;
   }
 
+  // labs_transcripts_gong_full_storage FF enables storing all Gong transcripts in a single datasource view
+  let gongFullStorage = false;
+  let gongFullStorageDataSourceViewId = null;
+  if (transcriptsConfiguration.provider === "gong") {
+    const featureFlags = await getFeatureFlags(owner);
+    gongFullStorage = featureFlags.includes(
+      "labs_transcripts_gong_full_storage"
+    );
+
+    if (gongFullStorage) {
+      const defaultTranscriptConfiguration =
+        await LabsTranscriptsConfigurationResource.fetchFirstConfigurationWithDatasourceViewForWorkspace(
+          auth
+        );
+
+      if (!defaultTranscriptConfiguration) {
+        localLogger.error(
+          {},
+          "[processTranscriptActivity] No default transcript configuration with a datasource view found while full Gong storage if enabled. Stopping."
+        );
+        await stopRetrieveTranscriptsWorkflow(transcriptsConfiguration);
```

Comment:
Wait should this be blocking the whole transcript workflow?

Reply:
Ok, not stopping workflow indeed